app/controllers/likes_controller.rb

```diff
@@ -7,9 +7,9 @@ def create
     )
     new_like.update_likes_counter
     if new_like.save
-      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", notice: 'Success!'
+      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", flash[:notice] = 'Your Like is saved!'
     else
-      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", alert: 'Error occured!'
+      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", flash.now[:error] = 'Could not save like'
```

Comment:
- [ ] Same here. While creating a like, I'm getting the error of `no implicit conversion of symbol into string`. Please fix this.

![Screenshot from 2022-02-25 22-58-35](https://user-images.githubusercontent.com/22434660/155761139-9f1e8e1e-968e-44a9-9bc9-ca53f189ea8a.png)

Reply:
Thanks for your time and feedback. I'll fix it ASAP.